2025-10-28 18:05:19,680 - Servidor Flask iniciado...
2025-10-28 18:05:59,551 - Monitoramento iniciado.
2025-10-28 18:08:59,822 - Erro ao obter preço do BTC: HTTPSConnectionPool(host='api.coingecko.com', port=443): Max retries exceeded with url: /api/v3/simple/price?ids=bitcoin&vs_currencies=brl (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000022BFDD7B6A0>: Failed to resolve 'api.coingecko.com' ([Errno 11001] getaddrinfo failed)"))
2025-10-28 18:09:00,768 - Erro ao obter preço do BTC: HTTPSConnectionPool(host='api.coingecko.com', port=443): Max retries exceeded with url: /api/v3/simple/price?ids=bitcoin&vs_currencies=brl (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000022BFDD7BB20>: Failed to resolve 'api.coingecko.com' ([Errno 11001] getaddrinfo failed)"))
2025-10-28 18:45:32,567 - Servidor Flask iniciado...
2025-10-28 18:45:33,524 - Servidor Flask iniciado...
2025-10-28 18:47:56,837 - Servidor Flask iniciado...
